packages/documentation/src/pages/getting-started/workflow/deploy.mdx

```diff
@@ -39,7 +39,17 @@ From here, Jenkins can kickoff a production deployment. After the deployment occ
 
 ## Rollbacks
 
-If a production deployment introduces issues that affect Service Level Objectives (SLOs) established for the project, the DevOps team may restore service to users by rolling back the deployment. This is accomplished by triggering a new deploy job in Jenkins using a previous release tag.
+If a production deployment introduces issues that affect Service Level Objectives (SLOs) 
+established for the project, you may restore service to users by rolling back 
+to a previous deployment. This is accomplished by triggering a new deploy job in Jenkins using a 
+previous release tag.
```

Comment:
it might be helpful to include context about expected time until deployed to prod, and how this is different from other deployment approaches & pipelines

Reply:
updated this